spec/acts_as_taggable_on/dirty_spec.rb

```diff
@@ -14,19 +14,27 @@
       end
 
       it 'should show changes of dirty object' do
-        expect(@taggable.changes).to eq({'tag_list' => ['awesome, epic', ['one']]})
+        expect(@taggable.changes).to eq({'tag_list' => [['awesome', 'epic'], ['one']]})
```

Comment:
⚠️ 

This is a breaking change. 

Either rollback your change or we have to communicate crazy about that. Our own internal code depends on the previous way it was formatted, and I bet a bunch of projects have the same.

Reply:
@Erowlin this PR is tagged for release 6.0.0, which is supposed to have breaking changes, no?

Reply:
Then I guess we're fine!

Reply:
I think this is a major breaking change that we should avoid. 
People expect to have tags in the root of the array not inside another array. 
Can we try to avoid this ?

Reply:
I can not promise I will have any time to invest into it in next days/weeks. Even though new behaviour seems much more logical to me, because previously it was a change from string to array, and now it's change from array to array ( no akward change of data type).